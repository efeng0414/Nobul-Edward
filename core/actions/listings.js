import {
  LISTINGS_IS_LOADING,
  MASTERLIST_IS_LOADING,
  GET_MASTERLIST_SUCCESS,
  GET_MASTERLIST_FAILURE,
  GET_SINGLE_LISTING_DETAILS_SUCCESS,
  GET_SINGLE_LISTING_DETAILS_FAILURE,
  GET_MULTIPLE_LISTING_DETAILS_SUCCESS,
  GET_MULTIPLE_LISTING_DETAILS_FAILURE,
  GET_FAVORITE_LISTINGS_DETAILS_SUCCESS,
  GET_FAVORITE_LISTINGS_DETAILS_FAILURE,
  GET_TAGGED_LISTINGS_DETAILS_SUCCESS,
  GET_TAGGED_LISTINGS_DETAILS_FAILURE,
  GET_DETAILS_AND_FEATURE_IMAGES_SUCCESS,
  GET_DETAILS_AND_FEATURE_IMAGES_FAILURE,
  FILTER_LISTINGS,
  SORT_LISTINGS_IDS_LOADING,
  SORT_LISTINGS_IDS_SUCCESS,
  SORT_LISTINGS_IDS_FAILURE
} from "../types/listings";

export const listingsIsLoading = isLoading => ({
  type: LISTINGS_IS_LOADING,
  payload: isLoading
});

export const masterlistIsLoading = isLoading => ({
  type: MASTERLIST_IS_LOADING,
  payload: isLoading
});

export const getMasterlistSuccess = (masterlist, provinceOrState) => ({
  type: GET_MASTERLIST_SUCCESS,
  payload: {
    masterlist,
    provinceOrState
  }
});

export const getMasterlistFailure = error => ({
  type: GET_MASTERLIST_FAILURE,
  payload: error
});

export const getSingleListingDetailsSuccess = data => ({
  type: GET_SINGLE_LISTING_DETAILS_SUCCESS,
  payload: data
});

export const getSingleListingDetailsFailure = data => ({
  type: GET_SINGLE_LISTING_DETAILS_FAILURE,
  payload: data
});

export const getMultipleListingDetailsSuccess = data => ({
  type: GET_MULTIPLE_LISTING_DETAILS_SUCCESS,
  payload: data
});

export const getMultileListingDetailsFailure = data => ({
  type: GET_MULTIPLE_LISTING_DETAILS_FAILURE,
  payload: data
});

export const getDetailsAndFeatureImagesSuccess = data => ({
  type: GET_DETAILS_AND_FEATURE_IMAGES_SUCCESS,
  payload: data
});

export const getDetailsAndFeatureImagesFailure = error => ({
  type: GET_DETAILS_AND_FEATURE_IMAGES_FAILURE,
  payload: error
});

export const filterListings = filter => ({
  type: FILTER_LISTINGS,
  payload: filter
});

export const getFavoriteListingsDetailsSuccess = data => ({
  type: GET_FAVORITE_LISTINGS_DETAILS_SUCCESS,
  payload: data
});

export const getFavoriteListingsDetailsFailure = data => ({
  type: GET_FAVORITE_LISTINGS_DETAILS_FAILURE,
  payload: data
});

export const getTaggedListingsDetailsSuccess = data => ({
  type: GET_TAGGED_LISTINGS_DETAILS_SUCCESS,
  payload: data
});

export const getTaggedListingsDetailsFailure = data => ({
  type: GET_TAGGED_LISTINGS_DETAILS_FAILURE,
  payload: data
});

export const sortListingsIdsSuccess = listingIds => ({
  type: SORT_LISTINGS_IDS_SUCCESS,
  payload: listingIds
});

export const sortListingsIdsFailure = error => ({
  type: SORT_LISTINGS_IDS_FAILURE,
  payload: error
});

export const sortListingsIdsLoading = error => ({
  type: SORT_LISTINGS_IDS_LOADING,
  payload: error
});
