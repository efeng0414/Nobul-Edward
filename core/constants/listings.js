export const RESIDENTIAL = "Residential";
export const COMMERCIAL = "Commercial";
export const HOUSE_TOWNHOUSE = "House/Townhouse";
export const CONDO_APARTMENT = "Condo/Apartment";
export const RECREATIONAL = "Recreational";
export const OTHER = "Other";

//TREB
export const RESIDENTIAL_PROPERTY = "ResidentialProperty";
export const CONDO_PROPERTY = "CondoProperty";
export const COMMERCIAL_PROPERTY = "CommercialProperty";

export const DEFAULT_PHOTO_COUNT = 25;

export const LISTING_SORT_OPTIONS = [
  "listTimeHighLow",
  "listTimeLowHigh",
  "listPriceLowHigh",
  "listPriceHighLow",
  "lotSizeRawLowHigh",
  "lotSizeRawHighLow",
  "lotFrontageHighLow",
  "lotFrontageLowHigh"
];
