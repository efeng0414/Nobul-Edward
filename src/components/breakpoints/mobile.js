import React from "react";
import PropTypes from "prop-types";

import Devices from "./devices";
import { MOBILE } from "../../../core/constants/breakpoints";

const Mobile = ({ children }) => <Devices sizes={[MOBILE]}>{children}</Devices>;

Mobile.propTypes = {
  children: PropTypes.any
};

export default Mobile;
