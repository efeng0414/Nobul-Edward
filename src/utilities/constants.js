import {
  OFFER_OPEN,
  OFFER_ACCEPTED,
  OFFER_NOT_ACCEPTED,
  OFFER_REJECTED,
  OFFER_JOBDELETED,
  OFFER_JOBEXPIRED,
  OFFER_WITHDRAWN
} from "../../core/constants/offers";

import { SELL, BUY } from "../../core/api-transform/jobs";
import { JOB_CLOSED } from "../../core/constants/jobs";

import {
  RESIDENTIAL as RESIDENTIAL_PROPERTY,
  COMMERCIAL as COMMERCIAL_PROPERTY,
  HOUSE_TOWNHOUSE as HOUSE_TOWNHOUSE_PROPERTY,
  CONDO_APARTMENT as CONDO_APARTMENT_PROPERTY,
  RECREATIONAL as RECREATIONAL_PROPERTY,
  OTHER as OTHER_PROPERTY
} from "../../core/constants/listings";

import {
  COMMERCIAL_BUY_ICON,
  COMMERCIAL_SELL_ICON,
  COMMERCIAL_GREY_ICON,
  CONDO_BUY_ICON,
  CONDO_SELL_ICON,
  CONDO_GREY_ICON,
  HOUSE_SELL_ICON,
  HOUSE_BUY_ICON,
  HOUSE_GREY_ICON,
  OTHER_BUY_ICON,
  OTHER_SELL_ICON,
  OTHER_GREY_ICON,
  RECREATIONAL_BUY_ICON,
  RECREATIONAL_SELL_ICON,
  RECREATIONAL_GREY_ICON,
  AVATAR_DEFAULT_LARGE,
  AVATAR_MALE_LARGE,
  AVATAR_FEMALE_LARGE,
  AVATAR_DEFAULT_SMALL,
  AVATAR_DEFAULT_MEDIUM,
  AVATAR_MALE_SMALL,
  AVATAR_MALE_MEDIUM,
  AVATAR_FEMALE_SMALL,
  AVATAR_FEMALE_MEDIUM,
  AVATAR_DEFAULT_SELECTED,
  AVATAR_MALE_SELECTED,
  AVATAR_FEMALE_SELECTED
} from "./images";

import ConsumerRegistrationForm from "../components/consumer-registration-steps/registration-form";
import ConsumerRegistrationConfirm from "../components/consumer-registration-steps/confirm";
import {
  AVATAR_TYPE_DEFAULT,
  AVATAR_TYPE_MALE,
  AVATAR_TYPE_FEMALE,
  AVATAR_DISPLAY,
  AVATAR_EDITING,
  AVATAR_SELECTED,
  AVATAR_SMALL
} from "../../core/constants/users";

export const LIMITED_LIST_DEFAULT_ELEMS = 3;

export const RESIDENTIAL = "residential";
export const COMMERCIAL = "commercial";
export const OTHER = "other";

export const HOME = "home";
export const INVESTMENT = "investment";

export const GOLD = "#ffc300";
export const WATER_BLUE = "#1283c6";

export const OFFER_STATUS_CLASSNAME = {
  [OFFER_OPEN]: "offer-status-open",
  [OFFER_ACCEPTED]: "offer-status-accepted",
  [OFFER_NOT_ACCEPTED]: "offer-status-no-accepted",
  [OFFER_REJECTED]: "offer-status-rejected",
  [OFFER_JOBDELETED]: "offer-status-job-deleted",
  [OFFER_JOBEXPIRED]: "offer-status-job-expired",
  [OFFER_WITHDRAWN]: "offer-status-withdrawn"
};

export const RegistrationStepMap = {
  0: ConsumerRegistrationForm,
  1: ConsumerRegistrationConfirm
};

export const HOME_PAGE_BUY_SELL_BG_CLASS = "blue";
export const HOME_PAGE_BROWSE_BG_CLASS = "purple";
export const HOME_PAGE_LEARN_BG_CLASS = "yellow";

export const CONUMER_REGISTRATION_STEP3_TITLE = [
  "consumer.contract_to_brokerage_title",
  "consumer.verification_required",
  "consumer.unverified_user_title"
];

export const CONUMER_REGISTRATION_STEP3_LINE1 = [
  "consumer.contract_to_brokerage_line1",
  "consumer.thank_you",
  "consumer.unverified_user_line_1"
];

export const CONUMER_REGISTRATION_STEP3_LINE2 = [
  "consumer.contract_to_brokerage_line2",
  "consumer.verification_send",
  "consumer.unverified_user_line_2"
];

export const PROPERTY_MAP = {
  [SELL]: {
    [RESIDENTIAL_PROPERTY]: HOUSE_SELL_ICON,
    [COMMERCIAL_PROPERTY]: COMMERCIAL_SELL_ICON,
    [HOUSE_TOWNHOUSE_PROPERTY]: HOUSE_SELL_ICON,
    [CONDO_APARTMENT_PROPERTY]: CONDO_SELL_ICON,
    [RECREATIONAL_PROPERTY]: RECREATIONAL_SELL_ICON,
    [OTHER_PROPERTY]: OTHER_SELL_ICON
  },
  [BUY]: {
    [RESIDENTIAL_PROPERTY]: HOUSE_BUY_ICON,
    [COMMERCIAL_PROPERTY]: COMMERCIAL_BUY_ICON,
    [HOUSE_TOWNHOUSE_PROPERTY]: HOUSE_BUY_ICON,
    [CONDO_APARTMENT_PROPERTY]: CONDO_BUY_ICON,
    [RECREATIONAL_PROPERTY]: RECREATIONAL_BUY_ICON,
    [OTHER_PROPERTY]: OTHER_BUY_ICON
  },
  [JOB_CLOSED]: {
    [RESIDENTIAL_PROPERTY]: HOUSE_GREY_ICON,
    [COMMERCIAL_PROPERTY]: COMMERCIAL_GREY_ICON,
    [HOUSE_TOWNHOUSE_PROPERTY]: HOUSE_GREY_ICON,
    [CONDO_APARTMENT_PROPERTY]: CONDO_GREY_ICON,
    [RECREATIONAL_PROPERTY]: RECREATIONAL_GREY_ICON,
    [OTHER_PROPERTY]: OTHER_GREY_ICON
  }
};

export const AVATAR_MAP = {
  [AVATAR_DISPLAY]: {
    [AVATAR_TYPE_DEFAULT]: AVATAR_DEFAULT_LARGE,
    [AVATAR_TYPE_MALE]: AVATAR_MALE_LARGE,
    [AVATAR_TYPE_FEMALE]: AVATAR_FEMALE_LARGE
  },
  [AVATAR_EDITING]: {
    [AVATAR_TYPE_DEFAULT]: AVATAR_DEFAULT_MEDIUM,
    [AVATAR_TYPE_MALE]: AVATAR_MALE_MEDIUM,
    [AVATAR_TYPE_FEMALE]: AVATAR_FEMALE_MEDIUM
  },
  [AVATAR_SMALL]: {
    [AVATAR_TYPE_DEFAULT]: AVATAR_DEFAULT_SMALL,
    [AVATAR_TYPE_MALE]: AVATAR_MALE_SMALL,
    [AVATAR_TYPE_FEMALE]: AVATAR_FEMALE_SMALL
  },
  [AVATAR_SELECTED]: {
    [AVATAR_TYPE_DEFAULT]: AVATAR_DEFAULT_SELECTED,
    [AVATAR_TYPE_MALE]: AVATAR_MALE_SELECTED,
    [AVATAR_TYPE_FEMALE]: AVATAR_FEMALE_SELECTED
  }
};

export const SERVICES_CHECKBOXES = "services-checkboxes";
export const SERVICES_GROUPS = "services-groups";
export const SERVICES_ACCORDION = "services-accordion";

export const PRICE_RANGE_SELECTOR_CLASS_NAME = "select-price-range";

export const locationSearchFilterOutTypes = [
  "room",
  "establishment",
  "finance",
  "food",
  "general_contractor",
  "place_of_worship",
  "street_number",
  "street_address",
  "subPremise",
  "natural_feature",
  "health"
];

export const minPasswordLength = 8;
export const maxPasswordLength = 16;

export const SELECT_CARD = "selectCard";
export const ADD_CARD = "addCard";

export const LANDING_PAGE_QUERY_PARAM = "ref=landing";
