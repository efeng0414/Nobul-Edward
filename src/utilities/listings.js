import { PRICE, DATE } from "../../core/constants/shared";

export const getListingCardDataFromAPIData = ({ APIData }) => {
  const {
    $: { ID: uid } = {
      ID: ""
    }
  } = APIData;
  const { featureImageUrl } = APIData;

  const { Price: [price] = [""] } = APIData;

  const { Address = [{}] } = APIData;
  const { City = [""] } = Address[0];
  const [location] = City;

  const { ListingID = [""] } = APIData;
  const [MLS] = ListingID;

  const { AgentDetails = [{}] } = APIData;
  const { Office = [{}] } = AgentDetails[0];
  const { Name = [] } = Office[0];
  const [brokerage] = Name;

  const { AddressLine1 = [""] } = Address[0];
  const [address] = AddressLine1;
  const { BedroomsTotal } = APIData.Building[0];
  const bedrooms = BedroomsTotal && parseInt(BedroomsTotal[0]);
  const { BathroomTotal } = APIData.Building[0];
  const bathrooms = BathroomTotal && parseInt(BathroomTotal[0]);

  return {
    uid,
    featureImageUrl,
    price,
    city: location, // lets call it 'city' because withRouter HOC uses a location prop
    bathrooms,
    bedrooms,
    MLS,
    brokerage,
    address
  };
};

export const listingSortMap = {
  [PRICE]: (prev, next) => parseFloat(prev.price) > parseFloat(next.price),
  [DATE]: (prev, next) => prev.lastUpdated > next.lastUpdated
};
