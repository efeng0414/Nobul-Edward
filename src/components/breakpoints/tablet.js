import React from "react";
import PropTypes from "prop-types";

import Devices from "./devices";
import { TABLET } from "../../../core/constants/breakpoints";

const Tablet = ({ children }) => <Devices sizes={[TABLET]}>{children}</Devices>;

Tablet.propTypes = {
  children: PropTypes.any
};

export default Tablet;
