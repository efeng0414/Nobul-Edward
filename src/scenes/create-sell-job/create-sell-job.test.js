import React from "react";
import renderer from "react-test-renderer";
import PropTypes from "prop-types";
import { Provider } from "react-redux";
import { IntlProvider, intlShape } from "react-intl";
import CreateSellJobWrapped from "./container";
import {
  SERVICES_EXACT,
  BATHROOMS,
  BEDROOMS,
  DESCRIPTION,
  REGIONS_FIELD
} from "../../../core/constants/shared";
import { MOBILE } from "../../../core/constants/breakpoints";
import { CONDO_APARTMENT } from "../../../core/constants/listings";

describe("Create Sell Job", () => {
  const containerProps = {
    intl: global.getIntl(),
    history: { location: {}, push: () => {} },
    jobs: { sell: {}, buy: {} },
    saveJob: () => {},
    users: { currentUser: { uid: "123" } },
    polygons: {},
    getAllConsumerSellJobs: () =>
      new Promise(resolve => {
        resolve();
      }),
    getPolygonBoundaries: () => {},
    currentBreakPoint: MOBILE
  };

  const containerState = {
    test: 0,
    currentStep: 1,
    currentScreen: 1,
    fields: {
      [BATHROOMS]: 0,
      [BEDROOMS]: 0,
      [SERVICES_EXACT]: [],
      [DESCRIPTION]: "",
      [REGIONS_FIELD]: {}
    },
    serviceRange: 3,
    selectedPropertyClass: "",
    showCongratulationsModal: false,
    showAnonymousUserModal: false,
    showLoginModal: false
  };

  const {
    WrappedComponent: CreateSellJob
  } = CreateSellJobWrapped.WrappedComponent.WrappedComponent;

  const mockStore = {
    subscribe: () => {},
    dispatch: () => {},
    getState: () => ({
      breakpoints: {
        currentBreakPoint: ""
      },
      authentication: {
        isLoading: false,
        currentUser: { uid: "123" }
      },
      anonymousEventListeners: {
        favoriteList: {}
      },
      jobs: {
        sell: {},
        buy: {}
      }
    })
  };

  const contextObject = {
    context: { intl: containerProps.intl, store: mockStore },
    childContextTypes: {
      intl: intlShape.isRequired,
      store: PropTypes.object
    }
  };
  const containerJSX = <CreateSellJob {...containerProps} />;

  const containerStoreJSX = (
    <Provider store={mockStore}>
      <CreateSellJob {...containerProps} />
    </Provider>
  );

  const containerShallow = global.Enzyme.shallow(containerJSX, contextObject);

  describe("Container Snapshot", () => {
    const snapshotJSX = (
      <IntlProvider {...global.intl}>{containerStoreJSX}</IntlProvider>
    );

    const containerSnapshot = renderer.create(snapshotJSX).toJSON();
    it("should match stored snapshot", () => {
      expect(containerSnapshot).toMatchSnapshot();
    });
  });

  describe("Container mount", () => {
    containerShallow.setProps({ scroll: () => ({}) });

    afterEach(() => {
      containerShallow.setState(containerState);
    });

    describe("determine screen count", () => {});

    describe("handle previous step", () => {
      const matchObject = {
        ...containerState
      };

      it("first screen", () => {});

      it("other screens", () => {});
    });

    describe("handle next step", () => {
      const matchObject = {
        ...containerState,
        fields: {
          priceRange: [0, 3],
          propertyType: CONDO_APARTMENT,
          [DESCRIPTION]: DESCRIPTION
        }
      };

      it("handleSaveJob is called", () => {
        containerShallow.setState({
          currentStep: 5,
          currentScreen: 1
        });

        const spy = jest.fn();
        Object.defineProperties(containerShallow.instance(), {
          handleSaveJob: {
            value: spy
          }
        });
        containerShallow.instance().handleNextStep(matchObject.fields);
        expect(spy).toHaveBeenCalled();
      });

      it("currentScreen equals numberOfScreens", () => {
        const stepObj = { currentStep: 3, currentScreen: 2 };

        containerShallow.setState({
          currentStep: stepObj.currentStep,
          currentScreen: stepObj.currentScreen
        });
        containerShallow.instance().handleNextStep(matchObject.fields);
        matchObject.currentStep = stepObj.currentStep + 1;
        matchObject.currentScreen = 1;
        matchObject.selectedPropertyClass = matchObject.fields.propertyType;
        expect(containerShallow.instance().state).toMatchObject(matchObject);
      });

      it("currentScreen not equals numberOfScreens", () => {
        const stepObj = { currentStep: 3, currentScreen: 1 };

        containerShallow.setState({
          currentStep: stepObj.currentStep,
          currentScreen: stepObj.currentScreen
        });
        containerShallow.instance().handleNextStep(matchObject.fields);
        matchObject.currentStep = stepObj.currentStep;
        matchObject.currentScreen = stepObj.currentScreen + 1;
        matchObject.selectedPropertyClass = matchObject.fields.propertyType;
        expect(containerShallow.instance().state).toMatchObject(matchObject);
      });
    });
  });
});
