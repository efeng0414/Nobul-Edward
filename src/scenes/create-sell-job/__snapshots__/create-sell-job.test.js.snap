// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Sell Job Container Snapshot should match stored snapshot 1`] = `
<div
  className="background null"
  style={
    Object {
      "backgroundColor": "",
      "backgroundImage": "url(file-mock)",
    }
  }
>
  <div
    className="background-content"
  >
    <div
      className="create-sell-job"
    >
      <div />
      <div
        className="ant-spin-nested-loading spinner-wrapper"
        style={null}
      >
        <div
          className="ant-spin-container"
        >
          <div
            className="create-sell-job__content"
          >
            <div
              className="create-sell-job__content--center"
            >
              <div
                className="sell-job-step1"
              >
                <form
                  className="ant-form ant-form-horizontal"
                  onSubmit={[Function]}
                >
                  <div
                    className=""
                  >
                    <h1>
                      My property is a:
                    </h1>
                    <div
                      className="ant-row ant-form-item"
                    >
                      <div
                        className="ant-form-item-control-wrapper"
                      >
                        <div
                          className="ant-form-item-control"
                        >
                          <span
                            className="ant-form-item-children"
                          >
                            <div
                              className="create-posting-property-checkbox-group"
                            >
                              <div
                                className="ant-radio-group ant-radio-group-outline"
                              >
                                <label
                                  className="ant-radio-wrapper"
                                >
                                  <span
                                    className="ant-radio"
                                    style={Object {}}
                                  >
                                    <input
                                      checked={false}
                                      className="ant-radio-input"
                                      disabled={false}
                                      onBlur={[Function]}
                                      onChange={[Function]}
                                      onFocus={[Function]}
                                      type="radio"
                                      value="House/Townhouse"
                                    />
                                    <span
                                      className="ant-radio-inner"
                                    />
                                  </span>
                                  <span>
                                    <div>
                                      <img
                                        className="propertyTypeIcon"
                                        src={
                                          Object {
                                            "default": "file-mock",
                                          }
                                        }
                                      />
                                    </div>
                                  </span>
                                </label>
                                <label
                                  className="ant-radio-wrapper"
                                >
                                  <span
                                    className="ant-radio"
                                    style={Object {}}
                                  >
                                    <input
                                      checked={false}
                                      className="ant-radio-input"
                                      disabled={false}
                                      onBlur={[Function]}
                                      onChange={[Function]}
                                      onFocus={[Function]}
                                      type="radio"
                                      value="Condo/Apartment"
                                    />
                                    <span
                                      className="ant-radio-inner"
                                    />
                                  </span>
                                  <span>
                                    <div>
                                      <img
                                        className="propertyTypeIcon"
                                        src={
                                          Object {
                                            "default": "file-mock",
                                          }
                                        }
                                      />
                                    </div>
                                  </span>
                                </label>
                                <label
                                  className="ant-radio-wrapper"
                                >
                                  <span
                                    className="ant-radio"
                                    style={Object {}}
                                  >
                                    <input
                                      checked={false}
                                      className="ant-radio-input"
                                      disabled={false}
                                      onBlur={[Function]}
                                      onChange={[Function]}
                                      onFocus={[Function]}
                                      type="radio"
                                      value="Recreational"
                                    />
                                    <span
                                      className="ant-radio-inner"
                                    />
                                  </span>
                                  <span>
                                    <div>
                                      <img
                                        className="propertyTypeIcon"
                                        src={
                                          Object {
                                            "default": "file-mock",
                                          }
                                        }
                                      />
                                    </div>
                                  </span>
                                </label>
                                <label
                                  className="ant-radio-wrapper"
                                >
                                  <span
                                    className="ant-radio"
                                    style={Object {}}
                                  >
                                    <input
                                      checked={false}
                                      className="ant-radio-input"
                                      disabled={false}
                                      onBlur={[Function]}
                                      onChange={[Function]}
                                      onFocus={[Function]}
                                      type="radio"
                                      value="Commercial"
                                    />
                                    <span
                                      className="ant-radio-inner"
                                    />
                                  </span>
                                  <span>
                                    <div>
                                      <img
                                        className="propertyTypeIcon"
                                        src={
                                          Object {
                                            "default": "file-mock",
                                          }
                                        }
                                      />
                                    </div>
                                  </span>
                                </label>
                                <label
                                  className="ant-radio-wrapper"
                                >
                                  <span
                                    className="ant-radio"
                                    style={Object {}}
                                  >
                                    <input
                                      checked={false}
                                      className="ant-radio-input"
                                      disabled={false}
                                      onBlur={[Function]}
                                      onChange={[Function]}
                                      onFocus={[Function]}
                                      type="radio"
                                      value="Other"
                                    />
                                    <span
                                      className="ant-radio-inner"
                                    />
                                  </span>
                                  <span>
                                    <div>
                                      <img
                                        className="propertyTypeIcon"
                                        src={
                                          Object {
                                            "default": "file-mock",
                                          }
                                        }
                                      />
                                    </div>
                                  </span>
                                </label>
                              </div>
                            </div>
                          </span>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="hide"
                  >
                    <h1>
                      I am hoping to sell for the following amount:
                    </h1>
                    <div
                      className="ant-row ant-form-item"
                    >
                      <div
                        className="ant-form-item-control-wrapper"
                      >
                        <div
                          className="ant-form-item-control has-success"
                        >
                          <span
                            className="ant-form-item-children"
                          >
                            <div
                              className="sell-job-step1-icon-slider-group"
                            >
                              <div
                                className="sell-job-step1-price-slider"
                              >
                                <div
                                  className="dual-price-range"
                                >
                                  <div
                                    className="ant-slider ant-slider-with-marks"
                                    onBlur={[Function]}
                                    onFocus={[Function]}
                                    onKeyDown={[Function]}
                                    onMouseDown={[Function]}
                                    onMouseUp={[Function]}
                                    onTouchStart={[Function]}
                                  >
                                    <div
                                      className="ant-slider-rail"
                                      style={Object {}}
                                    />
                                    <div
                                      className="ant-slider-track ant-slider-track-1"
                                      style={
                                        Object {
                                          "left": "8.333333333333332%",
                                          "width": "16.666666666666668%",
                                        }
                                      }
                                    />
                                    <div
                                      className="ant-slider-step"
                                    >
                                      <span
                                        className="ant-slider-dot ant-slider-dot-active"
                                        style={
                                          Object {
                                            "left": "8.333333333333332%",
                                          }
                                        }
                                      />
                                      <span
                                        className="ant-slider-dot ant-slider-dot-active"
                                        style={
                                          Object {
                                            "left": "25%",
                                          }
                                        }
                                      />
                                    </div>
                                    <div
                                      aria-disabled={false}
                                      aria-valuemax={12}
                                      aria-valuemin={0}
                                      aria-valuenow={1}
                                      className="ant-slider-handle ant-slider-handle-1"
                                      onBlur={[Function]}
                                      onKeyDown={[Function]}
                                      onMouseEnter={[Function]}
                                      onMouseLeave={[Function]}
                                      role="slider"
                                      style={
                                        Object {
                                          "left": "8.333333333333332%",
                                        }
                                      }
                                      tabIndex={0}
                                    />
                                    <div
                                      aria-disabled={false}
                                      aria-valuemax={12}
                                      aria-valuemin={0}
                                      aria-valuenow={3}
                                      className="ant-slider-handle ant-slider-handle-2"
                                      onBlur={[Function]}
                                      onKeyDown={[Function]}
                                      onMouseEnter={[Function]}
                                      onMouseLeave={[Function]}
                                      role="slider"
                                      style={
                                        Object {
                                          "left": "25%",
                                        }
                                      }
                                      tabIndex={0}
                                    />
                                    <div
                                      className="ant-slider-mark"
                                    >
                                      <span
                                        className="ant-slider-mark-text ant-slider-mark-text-active"
                                        onMouseDown={[Function]}
                                        onTouchStart={[Function]}
                                        style={
                                          Object {
                                            "left": "8.333333333333332%",
                                            "marginLeft": "-45%",
                                            "width": "90%",
                                          }
                                        }
                                      >
                                        $100,000
                                      </span>
                                      <span
                                        className="ant-slider-mark-text ant-slider-mark-text-active"
                                        onMouseDown={[Function]}
                                        onTouchStart={[Function]}
                                        style={
                                          Object {
                                            "left": "25%",
                                            "marginLeft": "-45%",
                                            "width": "90%",
                                          }
                                        }
                                      >
                                        $300,000
                                      </span>
                                    </div>
                                  </div>
                                </div>
                              </div>
                            </div>
                          </span>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    className="ant-row ant-form-item"
                  >
                    <div
                      className="ant-form-item-control-wrapper"
                    >
                      <div
                        className="ant-form-item-control"
                      >
                        <span
                          className="ant-form-item-children"
                        >
                          <div
                            className="sell-job-step1-buttons"
                          >
                            <button
                              className="ant-btn googletag-consumer-sell-real-estate-start ant-btn-primary ant-btn-lg"
                              onClick={[Function]}
                              type="submit"
                            >
                              <span>
                                Next
                              </span>
                            </button>
                          </div>
                        </span>
                      </div>
                    </div>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
